### ISSUE TITLE
Test on other datasets

### ISSUE THREAD

**Issue by LeiShenVictoria** [alexzhou907/dialogue_evaluation#6]
How to test (or finetune) your model on other datasets? Could you pls give an example of the command?
For evaluation of the "diversity" score, how to get a group of answers as you mentioned in README if we only have one answer generated by a model. Will the score be influenced a lot if we just feed one answer to calculate diversity?
Thx!

**Comment by WinnieHAN** [comment]
Yes, the score is influenced if only one answer is fed to calculate the diversity score. The n-gram based score is calculated based on a group of similar sentences. If only one sentence is considered, then the metric might be improper for diversity since the generated utterances given various queries are generally diverse. Thus we recommend using a set of augmented sentences with controlled paraphrasing.

**Comment by wizare** [comment]
I also meet the same issue. Can you tell me more about how to obtain the augmented sentences?
Thanks.
